tests/kubernetes/kubernetes_request_factory/test_pod_request_factory.py

```diff
@@ -62,7 +64,7 @@ def setUp(self):
                 'fsGroup': 2000,
             }
         )
-        self.maxDiff = None
+        self.maxDiff = None  # pylint: disable=invalid-name
```

Comment:
Why can't we rename this?

Reply:
It's nose's convention to keep maximum number of differences in the output

https://kite.com/python/docs/nose.case.Test.maxDiff

We can get rid of it if we switch to pytest.

Reply:
In this case it's https://docs.python.org/3/library/unittest.html#unittest.TestCase.maxDiff (not nose specific) and we won't be able to remove it until we stop subclassing unittest.TestCase (which can happen after pytest switch over, but not in the same commit)